cg/services/analysis_starter/configurator/implementations/raredisease.py

```diff
@@ -54,3 +92,8 @@ def _get_params_file_path(self, case_id: str) -> Path:
 
     def _get_work_dir(self, case_id: str) -> Path:
         return Path(self.root_dir, case_id, "work")
+
+    def _set_cluster_options(self, case_id: str) -> str:
```

Comment:
```suggestion
    def _get_cluster_options(self, case_id: str) -> str:
```
Seems to me to get more than set?